test-api-generator/citrus-test-api-generator-core/src/test/resources/org/citrusframework/openapi/generator/ExpectedCodeGenIT/expectedgen/rest/extpetstore/model/Pet.java

```diff
@@ -0,0 +1,268 @@
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.citrusframework.openapi.generator.rest.extpetstore.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Pet
+ */
+@jakarta.annotation.Generated(value = "org.citrusframework.openapi.generator.CitrusJavaCodegen", date = "2025-01-29T23:14:48.524898+01:00[Europe/Zurich]", comments = "Generator version: 7.9.0")
+public class Pet {
+  private Long id;
+
+  private String _name;
+
+  private Category category;
+
+  private List<String> photoUrls = new ArrayList<>();
+
+  private List<Tag> tags = new ArrayList<>();
+
+  /**
+   * pet status in the store
+   */
+  public enum StatusEnum {
+    AVAILABLE("available"),
+    
+    PENDING("pending"),
+    
+    SOLD("sold");
+
+    private String value;
+
+    StatusEnum(String value) {
+      this.value = value;
+    }
+
+    public String getValue() {
+      return value;
+    }
+
+    @Override
+    public String toString() {
+      return String.valueOf(value);
+    }
+
+    public static StatusEnum fromValue(String value) {
+      for (StatusEnum b : StatusEnum.values()) {
+        if (b.value.equals(value)) {
+          return b;
+        }
+      }
+      throw new IllegalArgumentException("Unexpected value '" + value + "'");
+    }
+  }
+
+  private StatusEnum status;
+
+  public Pet() {
+  }
+
+  public Pet id(Long id) {
+    
+    this.id = id;
+    return this;
+  }
+
+  /**
+   * Get id
+   * @return id
+   */
+  @jakarta.annotation.Nullable
+
+  public Long getId() {
+    return id;
+  }
+
+
+  public void setId(Long id) {
+    this.id = id;
+  }
+
+  public Pet _name(String _name) {
+    
+    this._name = _name;
+    return this;
+  }
+
+  /**
+   * Get _name
+   * @return _name
+   */
+  @jakarta.annotation.Nonnull
+
+  public String getName() {
+    return _name;
+  }
+
+
+  public void setName(String _name) {
+    this._name = _name;
+  }
+
+  public Pet category(Category category) {
+    
+    this.category = category;
+    return this;
+  }
+
+  /**
+   * Get category
+   * @return category
+   */
+  @jakarta.annotation.Nullable
+
+  public Category getCategory() {
+    return category;
+  }
+
+
+  public void setCategory(Category category) {
+    this.category = category;
+  }
+
+  public Pet photoUrls(List<String> photoUrls) {
+    
+    this.photoUrls = photoUrls;
+    return this;
+  }
+
+  public Pet addPhotoUrlsItem(String photoUrlsItem) {
+    if (this.photoUrls == null) {
+      this.photoUrls = new ArrayList<>();
+    }
+    this.photoUrls.add(photoUrlsItem);
+    return this;
+  }
+
+  /**
+   * Get photoUrls
+   * @return photoUrls
+   */
+  @jakarta.annotation.Nonnull
+
+  public List<String> getPhotoUrls() {
```

Comment:
strange newlines. but it's generated code, so I don't _really_ care 😉

Reply:
This is not even our code which generates the model files. Might be a matter of configuration though.